libs/stats/odc/stats/cli.py

```diff
@@ -30,10 +32,13 @@ def is_tile_in(tidx, tiles):
               default=None)
 @click.option('--year',
               type=int,
-              help="Only extract datasets for a given year")
+              help="Only extract datasets for a given year. This is a shortcut for --temporal-range=<int>--P1Y")
+@click.option('--temporal_range',
+              type=str,
+              help="Only extract datasets for a given time temporal_range, Example '2020-05--P1M' month of May 2020")
 @click.option('--period',
               type=str,
-              help="Only extract datasets for a given time period, Example '2020-05--P1M' month of May 2020")
+              help="Extract datasets for a given period specified, Example 'all', 'annual', '2020-03--P2M' ")
```

Comment:
so how does seasonal thing works?
Also is `period` a good name for this, maybe "frequency"?

Reply:
I've added an example to help string.  It'd be something like %Y-03--P2M. 

I also think frequency is more intuitive, so changed this accordingly.